### PR TITLE
qpdf 11.9.1 :snowflake:

### PR DESCRIPTION
**Pull request by boldorider4**
qpdf 11.9.1 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-6320]
- dev_url:        https://github.com/qpdf/qpdf/tree/v11.9.1
- conda_forge:    https://github.com/conda-forge/qpdf-feedstock
- pypi:           n/a
- pypi inspector: n/a

### Explanation of changes:

- new version number

[PKG-6320]: https://anaconda.atlassian.net/browse/PKG-6320?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by skupr-anaconda** [comment]
`libjpeg-turbo` will be incompatible with `jpeg` in the same environment, see https://github.com/AnacondaRecipes/libjpeg-turbo-feedstock/blob/master/recipe/meta.yaml#L26. We should be confident that clashes won't happen

**Comment by ifitchet** [comment]
l.34: `[build_platform == target_platform]` is not for us (probably)

**Comment by ifitchet** [comment]
I see all the `qpdf/qtest/*` tests are Perl scripts which is interesting -- is there a Python API?  I was wondering if there is any test we can run above and beyond `--help`.

**Comment by boldorider4** [comment]
> I see all the `qpdf/qtest/*` tests are Perl scripts which is interesting -- is there a Python API? I was wondering if there is any test we can run above and beyond `--help`.

I don't think there is. But I'll have a look at the qtests!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,27 +10,28 @@ source:
   sha256: 2ba4d248f9567a27c146b9772ef5dc93bd9622317978455ffe91b259340d13d1
 
 build:
-  number: 1
+  number: 0
+  # missing libjpeg-turbo for s390x
```
Comment (tombenes) [inline recipe/meta.yaml]:
```suggestion
  # missing jpeg for s390x
```
Still valid but it's `jpeg`
```
conda search jpeg --platform s390x -c sfe1ed40
Loading channels: done
No match found for: jpeg. Search: *jpeg*
```

Comment (skupr-anaconda) [reply recipe/meta.yaml]: `jpeg` exists on `linux-s390x`
```
conda search jpeg --platform linux-s390x
Loading channels: done
# Name                       Version           Build  Channel
jpeg                              9b      h2a837d6_2  repo/main
jpeg                              9d      h2a837d6_0  repo/main
jpeg                              9e      h2a837d6_0  repo/main
jpeg                              9e      hc33a586_1  repo/main
jpeg                              9e      hc33a586_2  repo/main
jpeg                              9e      hc33a586_3  repo/main
```

**Review by skupr-anaconda — APPROVED** [review]
LGTM but you can enable s390x because jpeg 9e is available there

File: recipe/meta.yaml (hunk 1 above)
Comment (tombenes) [reply recipe/meta.yaml]: Actually as mentioned by Serhii, it's available.
```
conda search jpeg --platform linux-s390x
Loading channels: done
# Name                       Version           Build  Channel
jpeg                              9b      h2a837d6_2  pkgs/main
jpeg                              9d      h2a837d6_0  pkgs/main
jpeg                              9e      h2a837d6_0  pkgs/main
jpeg                              9e      hc33a586_1  pkgs/main
jpeg                              9e      hc33a586_2  pkgs/main
jpeg                              9e      hc33a586_3  pkgs/main
```
Lesson learned - `conda search` requires full platform name

**Comment by tombenes** [comment]
We could also include those tests: https://github.com/qpdf/qpdf/blob/v11.9.1/pkg-test/README.md

**Review by tombenes — APPROVED** [review]

**Pull request merged**